backend/src/model/entity.rs

```diff
@@ -11,70 +11,224 @@ use diesel::{Identifiable, Insertable, Queryable};
+    pub edible_uses_en: Option<String>,
+    /// Fetched from PracticalPlants as \`medicinal_uses\` and merged with Permapeople. (fetched from PracticalPlants and Permapeople)
     pub medicinal_uses: Option<String>,
+    /// (fetched from PracticalPlants)
     pub material_uses_and_functions: Option<String>,
+    /// (fetched from PracticalPlants)
     pub botanic: Option<String>,
-    pub propagation: Option<String>,
-    pub cultivation: Option<String>,
-    pub environment: Option<String>,
+    /// (fetched from PracticalPlants)
     pub material_uses: Option<String>,
+    /// (fetched from PracticalPlants)
     pub functions: Option<String>,
-    pub provides_forage_for: Option<String>,
-    pub provides_shelter_for: Option<String>,
-    pub hardiness_zone: Option<i16>,
+    /// (fetched from PracticalPlants)
     pub heat_zone: Option<i16>,
-    pub water: Option<Water>,
-    pub sun: Option<Sun>,
+    /// (fetched from PracticalPlants)
     pub shade: Option<String>,
+    /// Merged between Permapeople and PracticalPlants. (fetched from PracticalPlants and Permapeople)
     pub soil_ph: Option<Vec<Option<SoilPh>>>,
+    /// Merged with soil_type of Permapeople. (fetched from PracticalPlants and Permapeople)
     pub soil_texture: Option<Vec<Option<SoilTexture>>>,
+    /// (fetched from PracticalPlants)
     pub soil_water_retention: Option<Vec<Option<SoilWaterRetention>>>,
+    /// (fetched from PracticalPlants)
     pub environmental_tolerances: Option<Vec<Option<String>>>,
-    pub native_climate_zones: Option<String>,
-    pub adapted_climate_zones: Option<String>,
+    /// (fetched from PracticalPlants)
     pub native_geographical_range: Option<String>,
+    /// (fetched from PracticalPlants)
     pub native_environment: Option<String>,
+    /// (fetched from PracticalPlants)
     pub ecosystem_niche: Option<String>,
-    pub root_zone_tendancy: Option<RootZoneTendancy>,
+    /// Fetched from PracticalPlants and merged with \`leaves\` of Permapeople. (fetched from PracticalPlants and Permapeople)
     pub deciduous_or_evergreen: Option<DeciduousOrEvergreen>,
+    /// (fetched from PracticalPlants)
     pub herbaceous_or_woody: Option<HerbaceousOrWoody>,
+    /// Merged with life_cycle of Permapeople. (fetched from PracticalPlants and Permapeople)
     pub life_cycle: Option<Vec<Option<LifeCycle>>>,
-    pub growth_rate: Option<GrowthRate>,
-    pub mature_size_height: Option<String>,
-    pub mature_size_width: Option<String>,
+    /// Merged with growth of Permapeople. (fetched from PracticalPlants and Permapeople)
+    pub growth_rate: Option<Vec<Option<GrowthRate>>>,
+    /// Fetched from PracticalPlants as \`mature_size_height\` and merged with Permapeople. (fetched from PracticalPlants and Permapeople)
+    pub height: Option<String>,
+    /// Fetched from PracticalPlants as \`mature_size_width\` and merged with Permapeople. (fetched from PracticalPlants and Permapeople)
```

Comment:
Units must be mentioned here (cm).
```suggestion
    /// Unit: Centimeter, Fetched from PracticalPlants as \`mature_size_width\` and merged with Permapeople. (fetched from PracticalPlants and Permapeople)
```

Reply:
I'm not sure if all the values have the same units e.g. there are `feet` values inside. the same challenge I described for the diversity of value formatting.